webpack/containers/Application/config.js

```diff
@@ -23,4 +25,8 @@ export const links = [
     path: 'subscriptions/:id(\[0-9]*$\)',
     component: SubscriptionDetails,
   },
+  {
+    path: 'xui/select_organization',
```

Comment:
What if we just call this /organization_select?  I would like to avoid putting things into /xui unless they are in the labs section.